install.sh

```diff
@@ -24,7 +24,7 @@ CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 ARCH="${ARCH/armv8l/armv7l}"
 
 # BOOTSTRAP_PACKAGES cannot depend on crew_profile_base for their core operations (completion scripts are fine)
-BOOTSTRAP_PACKAGES="pixz jq ca_certificates git gmp ncurses libyaml ruby openssl"
```

Comment:
Why remove `jq`? jq is used in `install.sh` (I think the `jq` in most installs is much older.)

Reply:
The preinstalled version works well with our `jq` arguments in `install.sh`